Base/Dockerfile

```diff
@@ -27,7 +27,9 @@ RUN apt-get update -qqy \
 # Selenium
 #==========
 RUN  mkdir -p /opt/selenium \
-  && wget --no-verbose https://selenium-release.storage.googleapis.com/2.53/selenium-server-standalone-2.53.0.jar -O /opt/selenium/selenium-server-standalone.jar
+  && wget --no-verbose https://selenium-release.storage.googleapis.com/2.53/selenium-server-standalone-2.53.0.jar -O /opt/selenium/selenium-server-standalone-2.53.0.jar \
```

Comment:
This is intentional - to allow the user to control which version is used. We wouldn't want to download many versions, but it would allow us for example to include beta versions.